src/OagBundle/Resources/public/css/main.css

```diff
@@ -52,4 +52,30 @@ body {
 
 .ui-tooltip {
     white-space: pre-line;
-}
\ No newline at end of file
+}
+
+#existing-locations {
+    width: 100%;
+    height: 600px;
```

Comment:
This height setting is causing the map to overlap the first set of "Current" checkboxes for me.